### ISSUE TITLE
Dropout in RNNs

### ISSUE THREAD

**Issue by aymericdamien** [tflearn/tflearn#192]
It seems there is a bug where dropout in RNNs are still applied at prediction time, investigating if it is coming from tflearn or tensorflow.

**Comment by maxpagels** [comment]
I did a quick test and ran the sentiment analysis example [1] and Kera's equivalent [2] (using Tensorflow as a backend instead of Theano). Even with dropout, the Keras code doesn't have the same problem, which would suggest the bug is in tflearn.

[1]: https://github.com/tflearn/tflearn/blob/master/examples/nlp/lstm.py
[2]: https://github.com/fchollet/keras/blob/master/examples/imdb_lstm.py

**Comment by aymericdamien** [comment]
Hi, actually the error was that the TF wrapper always apply dropout, so it has to be modified with TFLearn 'is_training' op condition, to only apply dropout at training time.
